### ISSUE TITLE
Discuss future of `dataset` support with stakeholders

### ISSUE THREAD

**Issue by sdruskat** [citation-file-format/governance#4]
Alejandra Gonzalez-Beltran has got in touch with me to discuss the experimental dataset support and its relation with other (standard) formats and initiatives (W3C DCAT vocabulary, DataCite, RO-crate).

We should initiate this discussion and perhaps bring Arfon into it as well.

**Comment by jspaaks** [comment]
Do we have / can we get an overview of which proportion of `CITATION.cff` files use `type: dataset`?

**Comment by sdruskat** [comment]
> Do we have / can we get an overview of which proportion of `CITATION.cff` files use `type: dataset`?

Yes, I can run an analysis over the corpus I'm harvesting. Will take a bit of time.

**Comment by sdruskat** [comment]
Didn't take that long:

In the data for 15977 different repositories, I found 

- 327 instances where the `CITATION.cff` file included a line that starts with the string `type:` and has the string `dataset` in the same line for any version of the file I have on record.
- 321 instances where the `CITATION.cff` file included a line that starts with the string `type: dataset` for any version of the file I have on record.

Other files have no type information, or type information different than `dataset`, or are not UTF-8 decodable.

So given this simple metric, the ratio of repositories with a `dataset`-type CFF file at any given time is around `.02`.
